index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Samuel Felice | Full Stack Developer</title>
+</head>
+<body>
+    <header>
+        <span class="logo">My Logo</span> 
+        <div class="nav-icon">
+          <div class="icon"></div>
+          <div class="icon"></div>          
+          <div class="icon"></div>
+    </div>
+    </header>
+
+    <main>
+        <h1 class="primary-heading">I'm Samuel Glad to see you!</h1>
+
+        <p class="introductory-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <span class="lets-connect">LET’S CONNECT</span>
+
+        <div class="social-media-icons">
+            <img src="/icons/twitter.svg" alt="Twitter Logo" class="twitter">
+            <img src="/icons/linkedin.svg" alt="Linkedin Logo" class="linkedin">
+            <img src="/icons/medium.svg" alt="Medium Logo" class="medium">
+            <img src="/icons/github.svg" alt="Github Logo" class="github">
+            <img src="/icons/finger.svg" alt="Two Fingers" class="twofingers">
+        </div>
+    </main>

```

Comment:
- [ ] [REQUIRED] The list of icons should use a `<ul>` tag to group them, and `<li>` per icon; Also each icon should have a link (`<a>` tag). Please, wrap 🎁 the `<img>` tags in `<li>` & `<a>` tags. This way you can make your code more semantic for search engines and screen readers.

Reply:
As suggested by the previous reviewer, Kindly use the `<ul>` tag for your social media links, and put each image inside an `<li>` tag 😀✔